src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -127,6 +128,25 @@ def assert_openai_interactive_shell_launched(self):
             if not pattern in output:
                 raise CliTestError(f"Output from aks copilot did not contain '{pattern}'. Output:\n{output}")
 
+    def test_detect_az_error(self):
```

Comment:
Could you put these test cases to some other file? The test cases in `test_aks_commands.py` are live test cases.